environments/overlays/operate-first/pipelineruns/okd-coreos-all-4.12-pipelinerun.yaml

```diff
@@ -36,6 +36,8 @@ spec:
     - name: registry-credentials
       secret:
         secretName: okd-scos-robot-pull-secret
+    - name: s3-credentials
+      emptyDir: {}
```

Comment:
```suggestion
    - name: s3-credentials
      secret:
        secretName: okd-scos-aws-config
```

Reply:
I'm not sure we what to enable it right away, so mounting an emptyDir would be a good start